round4/team-14.md

```diff
@@ -0,0 +1,105 @@
+6. As someone who struggles with procrastination, I want an accountability feature that would make me spread out my work over multiple days so that I am not rushing to finish by the deadline.
+7. As someone with multiple things to do, I want a feature that will mark priority or urgency so that rewards and punishments scale with the importance of the task.
+8. As a student, I want the timer to work like a pomodoro so that I can factor in breaks and not burn out.
+9. As an animal-lover, I want food/treat mechanics for when I complete objectives so that I can use positive reinforcement for continuing to focus.
+10. As someone who needs flexibility, I want to be able to change the length of the timer for focus and breaks so that I can create long and short focus sessions.
+11. As a student who studies for extended periods of time, I want an option to take long breaks when I have completed a certain number of objectives
+12. As an animal enthusiast, I would like to level up or grow my pet for positive motivation for working.
+13. As someone who works late nights, I would like the option to toggle between light and dark mode so that I reduce my eyestrain.
+14. As an educator, I would like for the mechanics to be child-friendly and fun so that I can introduce time-management to my younger students.
+15. As a student, I would like to be notified when I can take a break so that I am not constantly checking the timer when it is done.
+16. As a busy student, I would like to schedule study sessions in advance for certain tasks, and get notifications for when that time comes.
+17. As a user, I would like switch between the sidebar and timer badge to minimize distractions while working.
+18. As a student who has trouble focusing while doing homework, I want to set a daily time goal for me to do homework.
+19. As a student who has an exam coming up, I want the extension to keep me on track the day before.
+
+## Notes
+
+* Will potentially use the Pokémon concept instead of Tamagachi to have more sprite options.  Will be careful of not breaking copyright laws, where [fair media usage is outlined here](https://press.pokemon.com/en/Assets-Use-Terms).
+
+## References & Inspiration
+
+1. [Tamagochi](https://tamagotchi.com/)
+2. [Habitica](https://habitica.com/)
+3. [pomofocus](https://pomofocus.io/app)
+4. [forest app](https://forestapp.cc/)
+
+## Technical Details
+
+### User Interface
+
+When users open the Chrome Extension, a side bar will appear which specifies the user projects, their current pet, and the timer they will use for productivity. There will also be an option to switch from a side bar so the extension takes up less screen space. The popup sonsists of the timer and actions to modify the timer.
+
+### API, Libraries, and Frameworks
+
+* React
+* HTML
+* CSS
+* JavaScript
+
+### Data Storage
+
+Game mechanics (health of the pet) will be stored in a JSON file.
```

Comment:
@ashfujiyama The Chrome Extension doesn't have access to the file system, it operates entirely in the browser. You can't store a JSON file—the only way to save a JSON file in the file system is to trigger a Download and have the user specify a path to save a file in.

What you need instead is to use either of the browser-based storage. You probably just need Local Storage, but if you want more sophisticated options (like, for instance, the ability to have multiple pets perhaps), you can use IndexedDB.

You can read more about the latter here:
https://blog.bitsrc.io/how-to-use-indexeddb-a-nosql-db-on-the-browser-f845da3caf35